doc/source/data/key-concepts.rst

```diff
@@ -36,75 +37,72 @@ This flexibility is a unique characteristic of Ray Datasets.
 Reading Data
 ============
 
-Datasets uses Ray tasks to read data from remote storage. When reading from a file-based datasource (e.g., S3, GCS), it creates a number of read tasks proportional to the number of CPUs in the cluster. Each read task reads its assigned files and produces an output block:
+Datasets uses Ray tasks to read data from remote storage. Each read task reads its assigned files and produces an output block:
```

Comment:
Nit: Not sure if "assigned" is the right term here, since it will probably create some not-so-useful questions in the user's mind - who "assigns" the files to a task, do I need to worry about doing that, etc.

Maybe something like "Each read task reads one or more files and produces an output block"? Also I feel like we should call out that this is a parallel read.

```suggestion
Datasets uses Ray tasks to read data from remote storage in parallel. Each read task reads one or more files and produces an output block:
```